src/main/java/teammates/common/util/StringHelper.java

```diff
@@ -134,13 +140,25 @@ public static String encrypt(String value) {
         }
     }
 
-    public static String decrypt(String message) {
+    /*
+     * Decrypts the supplied string.
+     *
+     * @param message the ciphertext as a hexadecimal string
+     * @return the plaintext
+     * @throws InvalidParameterException if the ciphertext is invalid.
+     * @throws RuntimeException if the decryption fails for any other reason, such as {@code Cipher} initialization failure.
+     */
+    public static String decrypt(String message) throws InvalidParametersException {
         try {
             SecretKeySpec sks = new SecretKeySpec(hexStringToByteArray(Config.ENCRYPTION_KEY), "AES");
             Cipher cipher = Cipher.getInstance("AES");
             cipher.init(Cipher.DECRYPT_MODE, sks);
             byte[] decrypted = cipher.doFinal(hexStringToByteArray(message));
             return new String(decrypted);
+        } catch (NumberFormatException | IllegalBlockSizeException | BadPaddingException e) {
+            log.warning("Attempted to decrypt invalid ciphertext: " + message);
+            // potential PMD bug? No stack trace info is lost here
+            throw new InvalidParametersException(e.getMessage()); //NOPMD
```

Comment:
I think `e.getMessage()` doesn't contain stack trace. Is it possible to do `InvalidParametersException(e)` instead?